raddoseLib.py

```diff
@@ -98,31 +113,35 @@ def rd3d_calc(flux=3.5e12, energy=12.66,
     """
     
     rd3d_dir = "rd3d"
-    inputFileName = "rd3d_input.txt"
-    outputFileName = "rd3d_Summary.csv"
+    inputFileName = f"rd3d_{os.getpid()}_input.txt"
+    outputFileName = f"rd3d_{os.getpid()}_Summary.csv"
 
     templateFilePath=os.path.join(rd3d_dir,templateFileName)
     inputFilePath=os.path.join(rd3d_dir,inputFileName)
     outputFilePath=os.path.join(rd3d_dir,outputFileName)
     
     copyfile(templateFilePath, inputFilePath)
-        
-    replaceLine(inputFilePath,"FLUX",'FLUX {:.2e}\n'.format(flux))
-    replaceLine(inputFilePath,"ENERGY",'ENERGY {:.2f}\n'.format(energy))
-    replaceLine(inputFilePath,"TYPE GAUSSIAN",'TYPE {:s}\n'.format(beamType))
-    replaceLine(inputFilePath,"FWHM",'FWHM {:.1f} {:.1f}\n'.format(fwhmX,fwhmY))
-    replaceLine(inputFilePath,"COLLIMATION",'COLLIMATION RECTANGULAR {:.1f} {:.1f}\n'.format(collimationX,collimationY))
-    replaceLine(inputFilePath,"WEDGE",'WEDGE 0 {:0.1f}\n'.format(wedge))
-    replaceLine(inputFilePath,"EXPOSURETIME",'EXPOSURETIME {:0.3f}\n'.format(exposureTime))
-    replaceLine(inputFilePath,"TRANSLATEPERDEGREE",
-                'TRANSLATEPERDEGREE {:0.4f} {:0.4f} {:0.4f}\n'.format(translatePerDegX,translatePerDegY,translatePerDegZ))
-    replaceLine(inputFilePath,"DIMENSION",'DIMENSION {:0.1f} {:0.1f} {:0.1f}\n'.format(dimX,dimY,dimZ))
-    replaceLine(inputFilePath,"PIXELSPERMICRON",'PIXELSPERMICRON {:0.1f}\n'.format(pixelsPerMicron))
-    replaceLine(inputFilePath,"ANGULARRESOLUTION",'ANGULARRESOLUTION {:0.1f}\n'.format(angularResolution))
-    replaceLine(inputFilePath,"STARTOFFSET",
-                'STARTOFFSET {:f} {:f} {:f}\n'.format(startOffsetX,startOffsetY,startOffsetZ))    
-    
-    out = run_rd3d(inputFilePath)
+
```

Comment:
I wonder whether this file is templated heavily enough that we should just use jinja2?